src/tasks/tasks.py

```diff
@@ -1389,6 +1393,86 @@ def collect_metrics(ix_to_tag_dict, tag_group):
         return collected_metrics
 
 
+@register_task("winogender-diagnostic", rel_path="RTE/diagnostics", n_classes=2)
+class WinogenderTask(GLUEDiagnosticTask):
+    """Supported wnogender task """
```

Comment:
Typo. Also, why 'Supported'?

Reply:
There's also another DNC task that's dnc-recast-winogender, which is not completely supported.